api/operator/v1/vmextra_types.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EntitySelectors namespace and object selector
+type EntitySelectors struct {
```

Comment:
Any reason to have it at `api` package? It's not used by any api entities.

Reply:
initially was going to use it for vmanomaly, but then decided to not introduce separate CRD configurations due to complexity of configuration structure